### DISCUSSION TITLE
Git error when rendering Manuscript Quarto documents

### DISCUSSION THREAD

**Discussion by SaoirseKelleher** [quarto-dev/quarto-cli#10694] [Q&A] [answered]
### Description

Hi, I've recently been having some issues with rendering the Manuscript document type in Quarto. Upon hitting the Render button (to html), I get the following error in the background jobs pane. I'm a bit confused why Quarto is trying to access git at all - this is a run-of-the-mill document with only text for now, and no code. The error is somewhat intermittent, restarting and rendering works occasionally, so I'm a bit stuck on even getting a reprex for this. This is on an Intel Mac, R 4.4.0, Quarto 1.5.56.

> ERROR: There is an error while retrieving information from remote 'origin'.
>  Git error: Connection closed by 103.12.108.13 port 22
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.
> . 
>  Git status code: 128.
> 
> Stack trace:
>  Git error: Connection closed by 103.12.108.13 port 22
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.
> . 
>  Git status code: 128.
>     at gitHubContext (file:///Applications/quarto/bin/quarto.js:73307:31)
>     at eventLoopTick (ext:core/01_core.js:153:7)
>     at async Object.filterParams (file:///Applications/quarto/bin/quarto.js:95426:39)
>     at async projectFilterParams (file:///Applications/quarto/bin/quarto.js:44243:45)
>     at async filterParamsJson (file:///Applications/quarto/bin/quarto.js:44015:12)
>     at async runPandoc (file:///Applications/quarto/bin/quarto.js:69675:24)
>     at async renderPandoc (file:///Applications/quarto/bin/quarto.js:70717:26)
>     at async Object.onRender (file:///Applications/quarto/bin/quarto.js:95075:33)
>     at async renderFileInternal (file:///Applications/quarto/bin/quarto.js:78245:17)
>     at async renderFiles (file:///Applications/quarto/bin/quarto.js:78069:17)

My _quarto.yml file for this project is as follows:

> project:
>   type: manuscript
> 
> manuscript:
>   article: sbPaper.qmd
> 
> format:
>   html:
>     theme: minty
>   docx: default
> 
> execute:
>   freeze: true

Any ideas?

**Comment by mcanouil** [comment, marked as answer]
This is a Git error unrelated.
You have extensive results/answers regarding this online.

The remote you set is either protected (private and you did not set credentials) or no longer exist.

FYI, see below in the context of Manuscript project type:

- https://github.com/quarto-dev/quarto-cli/issues/10031

**Comment by shawcmat** [comment]
I am also having this issue. My git credentials are correct, I have full access to my remote repository, and the issue comes and goes at random. Why is quarto having anything to do with git at all?

**Reply by mcanouil** [reply]
Please read the comment marked as answer, especially the link in it.
